### DISCUSSION TITLE
More compact view of device list

### DISCUSSION THREAD

**Discussion by HaArD2000** [File-New-Project/EarTrumpet#774] [Ideas]
I have multiple monitors that are recognized as Audio devices, plus wireless headphones and sound outputs. This makes for a long list of audio devices which cannot even fit on the screen, I have to scroll up and down to see them all.

It would be great to have a "compact" view that tightened up this display, perhaps hiding the System Sounds that appears below with every device until you select that device.  Also, smaller fonts/icons and less white space.

![EarTrumpetPic](https://user-images.githubusercontent.com/32270210/109397552-32434980-7905-11eb-816e-d69110e2e03f.png)

**Comment by riverar** [comment]
Do you use all those devices? If not, I recommend disabling a few of them via Windows Legacy > Playback Devices.

**Reply by HaArD2000** [reply]
Yes, I use them all.

**Comment by fheck** [comment]
Maybe a possibility would be to minimize each device to its master volume. That way the list gets shorter but the devices still are enabled.

**Reply by HaArD2000** [reply]
That could help, but it's not an option.

Every device shows a Master Volume and a Systems Sounds volume even thought the Systems Sounds are only actually being played on the device you direct them to. All the other entries are ghosts that don't really do anything.
